src/components/staking/Undelegations.vue

```diff
@@ -1,9 +1,21 @@
 <template>
-  <div v-if="undelegations && undelegations.length > 0" class="undelegations">
-    <h1>
-      Pending
-    </h1>
-    <TableUndelegations :undelegations="undelegations" />
+  <div class="table-container">
+    <div
+      v-if="$apollo.queries.undelegations.loading && !undelegations.length"
```

Comment:
you need a `undelegationsLoaded` property here. if no, this will show loading all the time when some update is triggered but you have no undelegations